content/spin/cache.md

```diff
@@ -0,0 +1,64 @@
+title = "Spin Cache"
+template = "spin_main"
+date = "2022-03-14T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/cache.md"
+
+---
+
+- [The Spin Registry Cache](#the-spin-registry-cache)
+- [Viewing Registry Cache Files](#viewing-registry-cache-files)
+- [Clearing Registry Cache Files](#clearing-registry-cache-files)
+
+## The Spin Registry Cache
+
+A running Spin application can [fetch resources from remote registries](https://developer.fermyon.com/spin/spin-oci) and individual component sources via HTTP endpoints. These resources naturally consume network bandwidth. To ensure network efficiency, and to prevent waiting to download the same files every time an application is started, Spin automatically maintains a local `spin/registry` cache.
+
+## Viewing Registry Cache Files
+
+All downloadable application files that are automatically cached can be found in the following file paths, depending on the operating system being used to run the Spin application:
+
+| Platform | Value | Example |
+| :--- | :--- | :--- |
+| Linux | `$XDG_CACHE_HOME` or `$HOME/.cache/spin/registry` | `/home/alice/.cache/spin/registry` |
+| macOS | `$HOME/Library/Caches/spin/registry` | `/Users/Alice/Library/Caches/spin/registry` |
+| Windows | `{FOLDERID_LocalAppData}/spin/registry` | `C:\Users\Alice\AppData\Local/spin/registry` |
+
+For example inspecting the local `spin/registry` cache on macOS, (using the `tree` command) would look like this:
+
+<!-- @selectiveCpy -->
+
+```console
+$ tree ~/Library/Caches/spin/registry/
+
+├── data
+│   ├── sha256:41a4649a8a8c176133792119cb45a7686767d3fa376ffd656e2ff76a6071fb07
+│   └── sha256:da3fda2db338a73483068072e22f7e7eef27afdbae3db824e130932adce703ba
+├── manifests
+│   └── ghcr.io
+│       └── radu-matei
+│           ├── hello-registries
+│           │   └── latest
+│           │       ├── config.json
+│           │       └── manifest.json
+│           └── spin-openai-demo
+│               └── v1
+│                   ├── config.json
+│                   └── manifest.json
+└── wasm
+    ├── sha256:0b985e7d43e719f34cbb54849759a2f8e7913c0f9b17bf7cb2b3d2458d33859e
+    └── sha256:d5f9e1f6b61b90f7404e3800285f7860fe2cfc7d0116023efc370adbb403fe87
+```
+
+The `data` directory contains all static assets referenced from applications distributed with remote registries. The `wasm` directory contains all component sources referenced either in applications distributed with remote registries, or component sources from HTTP endpoints, directly referenced in `spin.toml` (for example, the static file server, or the KV explorer). 
```

Comment:
Do we mention the KV explorer anywhere else?  Reading this para it feels like we expect the user to know what it is.

Reply:
Again, probably oversharing words. I will remove the content in the parenthesis.